docs/demos/coast-to-coast-q4-2023.md

```diff
@@ -0,0 +1,793 @@
+
+Verify this for the `Operator` with:
+```sh
+kubectl get operator/coastal -o yaml -w
+```
+
+and for the `Catalog` with:
+```sh
+kubectl get catalog/coastal -o yaml -w
+```
+
+Once the `Operator` has been successfully installed we can verify that all the resources created are healthy by checking the `BundleDeployment` resource owned by the `Operator` we created. Verify the `BundleDeployment` has a status condition showing whether or not it is healthy with:
+```sh
+kubectl get bundledeployment/coastal -o yaml -w
+```
+
+### Update the FBC Image to contain a bundle for `v1.1.0`
+To further highlight the version range selection, lets make sure that a `v1.1.0` bundle being
+added to the FBC image results in no upgrade occurring.
+
+- Overwrite the FBC YAML file to contain the previous content and the new bundle
+```sh
+cat << EOF > catalog/index.yaml
+{
+  "schema": "olm.package",
+  "name": "coastal"
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.0-alpha1",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.0-alpha1",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.0-alpha1"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.0.0",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.0.0",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.0.0"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.bundle",
+  "name": "coastal.v1.1.0",
+  "package": "coastal",
+  "image": "quay.io/operator-framework/coastal-bundle:v1.1.0",
+  "properties": [
+    {
+      "type": "olm.package",
+      "value": {
+        "packageName": "coastal",
+        "version": "1.1.0"
+      }
+    },
+    {
+      "type": "olm.bundle.mediatype",
+      "value": "plain+v0"
+    }
+  ]
+}
+{
+  "schema": "olm.channel",
+  "name": "stable",
+  "package": "coastal",
+  "entries": [
+    {
+      "name": "coastal.v1.1.0"
+    },
+    {
+      "name": "coastal.v1.0.0"
+    },
+    {
+      "name": "coastal.v1.0.0-alpha1"
+    }
+  ]
+}
+EOF
+```
+
+- Build and push the FBC image
+```sh
+docker build -t quay.io/operator-framework/coastal-catalog:latest -f catalog.Dockerfile . && \
+docker push quay.io/operator-framework/coastal-catalog:latest
+```
+
+Similarly to the last step, we should see the `Catalog` update it's resolved reference
+but this time we expect that the `Operator` resource remains the same and does not 
+automatically upgrade to `v1.1.0`
+
+### Update the FBC Image to contain a bundle for `v1.0.1`
+Highlighting the version range functionality again, lets add the `v1.0.1` bundle to our catalog
+to ensure it automatically upgrades within the z-stream.
```

Comment:
```suggestion
and ensure it automatically upgrades within the z-stream.
```

Reply:
I think you want _and_ here, correct? Because you can add a specific bundle, and whether or not it automatically updates depends on how you set things up, right?

Reply:
[08e8dbb](https://github.com/operator-framework/operator-controller/pull/533/commits/08e8dbba77679d610d04f2b8002613fe94421aff)